modules/skybrowser/include/screenspaceskybrowser.h

```diff
@@ -49,11 +49,13 @@ class ScreenSpaceSkyBrowser : public ScreenSpaceRenderable, public WwtCommunicat
     float opacity() const;
     glm::dvec2 fineTuneVector(const glm::dvec2& drag);
     bool isInitialized() const;
+    bool pointSpaceCraft() const;
```

Comment:
I'd personally say `pointSpacecraft`, since we don't write AirCraft we oughtn't say SpaceCraft imho